components/tikv_alloc/Cargo.toml

```diff
@@ -29,7 +29,7 @@ features = ["unprefixed_malloc_on_supported_platforms", "stats"]
 [dependencies.tikv-jemalloc-sys]
 version = "0.4.0"
 optional = true
-features = ["stats"]
+features = ["stats", "debug"]
```

Comment:
should it enabled by default？

Reply:
No...It may be caused by merge. I will remove it